### DISCUSSION TITLE
How to add video?

### DISCUSSION THREAD

**Discussion by AngangGuo** [McShelby/hugo-theme-relearn#967] [Q&A] [answered]
I have an local .mp4 video and want to show it on the page.
How can I show this video on the page?

**Comment by McShelby** [comment, marked as answer]
Currently the theme does not contain a shortcode for this but you could fallback to [plain HTML](https://gohugo.io/getting-started/configuration-markup/#rendererunsafe) in your markdown. See [`video` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video).

Alternativley, you can add a dedicated `video` shortcode like https://github.com/martignoni/hugo-video (haven't tried it myself)
